data/reusables/repositories/two-dot-diff-comparison-example-urls.md

```diff
@@ -1 +1 @@
-For example, this URL uses the shortened seven-character SHA codes to compare commits `c3a414e` and `faf7c6f`: `https://github.com/github/linguist/compare/c3a414e..faf7c6f.`
+For example, this URL uses the shortened seven-character SHA codes to compare commits `c3a414e` and `faf7c6f`: `https://github.com/github/linguist/compare/c3a414e..faf7c6f`
```

Comment:
Could move it outside the code fence either
```suggestion
For example, this URL uses the shortened seven-character SHA codes to compare commits `c3a414e` and `faf7c6f`: `https://github.com/github/linguist/compare/c3a414e..faf7c6f`.
```

Reply:
True. I figured clipping it off the end makes it easier for oblivious drag copy + paste, but it might just my anecdotal use-case